eslint_plugin/eslint_plugin_annotations.js

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+/**
+ * @param {*} filePath path to the file containing the issue
+ * @param {*} issue
+ * @return {Object} returns an annotation object as specified here:
+ * https://developer.github.com/v3/checks/runs/#annotations-object
+ */
+function setupSingleAnnotation (filePath, issue) {
+  const path = filePath
+  const startLine = issue.line
+  const endLine = issue.endLine
+  let annotationLevel = 'notice'
+  if (issue.severity === 1) {
+    annotationLevel = 'warning'
+  } else if (issue.severity === 2) {
+    annotationLevel = 'failure'
+  }
```

Comment:
can severity ever be 0 or 3?

Reply:
0 equals "notices" and that is why annotationLevel at first is a notice.
But it can't be more than 2: https://github.com/eslint/eslint#configuration